File: Q10/main.c
#include<stdio.h>
int main(){
    int row,col;
    scanf("%d%d",&row,&col);

    int arr[row][col];

    for(int i=0;i<row;i++){
        for(int j=0;j<col;j++){
            scanf("%d",&arr[i][j]);
        }
    }
    for(int i=0;i<row;i++){
        int max = arr[0][i];
        for(int j=0;j<col;j++){
            if(arr[j][i]>max){
                max = arr[j][i];
            }
        }
        printf("%d\n",max);
    }
    return 0;
}

File: Q10/main.py
row = int(input())
col = int(input())

arr = []

for i in range(row):
    row_data = list(map(int, input().split()))
    arr.append(row_data)

for i in range(col):
    max_value = arr[0][i]

    for j in range(row):
        if arr[j][i] > max_value:
            max_value = arr[j][i]

    print(max_value, end=" ")